ddtrace/commands/ddtrace_run.py

```diff
@@ -105,6 +104,9 @@ def main():
 
     log.debug("program executable: %s", executable)
 
+    if os.path.basename(executable) == "uwsgi":
+        log.warning("ddtrace-run is not supported with uwsgi. Use manual configuration.")
```

Comment:
I'm wondering if there's a point to log a "warning" where it's obvious things are going to not work at all.

If a user runs its program with `ddtrace-run` it's because they **want** traces.

Is it better to have the user being notified right away and fix the problem, or fail silently and leave them to wonder for 2 hours why they don't see any trace… until they find out a WARNING in their log.

I'd rather raise an error here and add a huge upgrade note in reno.